__init__.py

```diff
@@ -36,29 +39,32 @@ def split_sentences(text):
     sents = [i.replace('~.~', '.') for i in sents]
     if sents[-1][-1] in '.!?':
         sents[-1] = sents[-1][:-1]
-    print(sents)
     return sents
 
-
 class DuckduckgoSkill(CommonQuerySkill):
-    # Only ones that make sense in
-    # <question_word> <question_verb> <noun>
-    question_words = ['who', 'whom', 'what', 'when']
-    # Note the spaces
-    question_verbs = [' is', '\'s', 's', ' are', '\'re',
-                      're', ' did', ' was', ' were']
-    articles = ['a', 'an', 'the', 'any']
-    start_words = ['is', 'and', 'a', 'of', 'if', 'the',
-                   'because', 'since', 'for', 'by', 'from',
-                   'when', 'between', 'who', 'was', 'in']
-    is_verb = ' is '
-    in_word = 'in '
 
     def __init__(self):
         super(DuckduckgoSkill, self).__init__()
+        self.is_verb = ' is '
+        self.in_word = 'in '
+        # get ddg specific vocab for intent match
+        fname = self.find_resource("DuckDuckGo.voc", res_dirname="locale")
+        temp = read_vocab_file(fname)
+        vocab = []
+        for item in temp:
+            vocab.append( " ".join(item) )
+        self.sorted_vocab = sorted(vocab, key=lambda x: (-len(x), x))
+
+        self.translated_question_words = self.translate_list("question_words")
+        self.translated_question_verbs = self.translate_list("question_verbs")
+        self.translated_articles = self.translate_list("articles")
+        self.translated_start_words = self.translate_list("start_words")
 
+    """
```

Comment:
Looks like you commented out the classmethod version and made it and instance method.  can the commented code be removed?

Reply:
Yup, done